cmd/gds/main.go

```diff
@@ -37,392 +38,479 @@ func main() {
+					},
 				},
-				cli.StringFlag{
-					Name:  "t, text",
-					Usage: "the text to include in the note",
+			},
+			{
+				Name:     "admin:notes-create",
+				Usage:    "create a new note associated with a VASP",
+				Category: "admin",
+				Action:   adminCreateNote,
+				Before:   initAdminClient,
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:    "id",
+						Aliases: []string{"i"},
+						Usage:   "the uuid of the VASP to associate the note with",
+					},
+					&cli.StringFlag{
+						Name:    "name",
+						Aliases: []string{"n"},
+						Usage:   "the name of the new note or existing note",
+					},
+					&cli.StringFlag{
+						Name:    "text",
+						Aliases: []string{"t"},
+						Usage:   "the text to include in the note",
+					},
+					&cli.StringFlag{
+						Name:    "file",
+						Aliases: []string{"f"},
+						Usage:   "read note text from file",
+					},
 				},
-				cli.StringFlag{
-					Name:  "f, file",
-					Usage: "read note text from file",
+			},
+			{
+				Name:     "admin:notes-update",
+				Usage:    "update an existing VASP note",
+				Category: "admin",
+				Action:   adminUpdateNote,
+				Before:   initAdminClient,
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:    "id",
+						Aliases: []string{"i"},
+						Usage:   "the uuid of the VASP the note is associated with",
+					},
+					&cli.StringFlag{
+						Name:    "name",
+						Aliases: []string{"n"},
+						Usage:   "the name of the note to update",
+					},
+					&cli.StringFlag{
+						Name:    "text",
+						Aliases: []string{"t"},
+						Usage:   "the text to include in the note",
+					},
+					&cli.StringFlag{
+						Name:    "file",
+						Aliases: []string{"f"},
+						Usage:   "read note text from file",
+					},
 				},
 			},
-		},
-		{
-			Name:     "admin:notes-delete",
-			Usage:    "delete an existing VASP note",
-			Category: "admin",
-			Action:   adminDeleteNote,
-			Before:   initAdminClient,
-			Flags: []cli.Flag{
-				cli.StringFlag{
-					Name:  "i, id",
-					Usage: "the uuid of the VASP the note is associated with",
+			{
+				Name:     "admin:notes-delete",
+				Usage:    "delete an existing VASP note",
+				Category: "admin",
+				Action:   adminDeleteNote,
+				Before:   initAdminClient,
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:    "id",
+						Aliases: []string{"i"},
+						Usage:   "the uuid of the VASP the note is associated with",
+					},
+					&cli.StringFlag{
+						Name:    "name",
+						Aliases: []string{"n"},
+						Usage:   "the name of the note to delete",
+					},
 				},
-				cli.StringFlag{
-					Name:  "n, name",
-					Usage: "the name of the note to delete",
+			},
+			{
+				Name:      "profile",
+				Aliases:   []string{"config"},
+				Usage:     "view and manage profiles to configure client with",
+				UsageText: "gds profile [name]\n   gds profile --activate [name]\n   gds profile --list\n   gds profile --path\n   gds profile --install",
```

Comment:
Is the idea that the user edits the configuration file themselves? Would there be any benefit to also allowing the user to set individual config settings through the CLI?